src/lib/songdata.ts

```diff
@@ -11,7 +11,7 @@ export const songData: Songs = {
 		},
 		{
 			artist: 'Jason Shaw',
-			title: 'SOUTH OF THE ORDER',
+			title: 'South of the order',
```

Comment:
The original title is in upper case, therefore i would keep it.

Reply:
done